### PR TITLE
Feat/137/new account verification design

### PR DESCRIPTION
**Pull request by VrushaliHaldankar01**
new account verification design
image
as per the figma design 

image
All changes should be tested and verified to work correctly.
Code follows frontend best practices.
Branch names follow the conventions in the contributing file.
Commit messages follow the naming conventions in the contributing file.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,4 +8,19 @@ API_URL=http://localhost:8000
 CLIENT_URL=http://localhost:3000
 JWT_EMAIL_VERIFICATION_SECRET=
 JWT_LIFETIME=
-JWT_EMAIL_LIFETIME=
\ No newline at end of file
+JWT_EMAIL_LIFETIME=
+
+
+# PORT=8000
+# MONGODB_URI=mongodb+srv://vrushhaldankar16:vrushali123@cluster0.j4lo0kc.mongodb.net/
```
Comment (AnnaSolovykh) [inline .env.example:15]: Vrushali, this is your personal data.

File: middleware/error-handler.js [hunk 2]
```diff
@@ -26,7 +26,9 @@ const errorHandlerMiddleware = (err, req, res, next) => {
   }
   if (err.name === 'TokenExpiredError') {
     customError.message =
-      err.userMessage || 'Session expired, please login again';
+      //err.userMessage || 'Session expired, please login again';
+      err.userMessage || 'jwt expired';
```
Comment (AnnaSolovykh) [inline middleware/error-handler.js:30]: Can you give a more user-friendly message to the user?  They don't know what jwt is, I bet you :)))

**Comment by AnnaSolovykh** [comment]
@VrushaliHaldankar01 excuse me what are the testing steps? I can't find this page, I am sorry.

**Comment by ElenaCherpakova** [comment]
@VrushaliHaldankar01  Your whole branch is outdated. Before opening a PR from your branch to dev, follow these steps:

1. Make sure you are on your branch:
   `git checkout your-branch`

2. Pull the latest changes from the dev branch:
   `git pull origin dev`

3. Resolve any merge conflicts if they arise.

4. Once your branch is updated, you can open a PR to the dev branch.

I was sharing this great image of Githib workflow in our group. 
Screen Shot 2023-08-22 at 2 30 12 PM

**Comment by VrushaliHaldankar01** [comment]
Yes ill try to resolve the issues asap,thank you

**Pull request closed without merging**